packages/cryptoassets/src/currencies.js

```diff
@@ -2698,6 +2726,36 @@ const cryptocurrenciesById: { [name: string]: CryptoCurrency } = {
       },
     ],
   },
+  crypto_org_croeseid: {
+    type: "CryptoCurrency",
+    id: "crypto_org_croeseid",
+    coinType: 1,
```

Comment:
@TamtamHero what's your input about this: should the testnet accounts be derivated on same path as mainnet and therefore have same address or should it be on coinType 1?

existing choices:
- bitcoin will use 0 and 1
- ethereum will use 60 for both (because it's conventionally what everyone do, they derives testnet on 60' too)
present choice

Reply:
The device app only supports derivations on `m/44'/394'` so it doesn't make sense to add a testnet coinType here, nobody will be able to use it 
https://github.com/LedgerHQ/app-cryptocom/blob/cro/app/Makefile#L34

Reply:
That is correct, the app only supports coin type 394.
I checked with the team, and it seems that even though it is specified to use 1' per spec, using 394' is still possible for croeseid addresses. Thanks I will modify the value accordingly